src/main/java/org/broadinstitute/hellbender/utils/reference/ReferenceUtils.java

```diff
@@ -49,6 +49,12 @@ public static SAMSequenceDictionary loadFastaDictionary( final File fastaDiction
         catch ( IOException e ) {
             throw new UserException.CouldNotReadInputFile("Error loading fasta dictionary file " + fastaDictionaryFile, e);
         }
+        catch ( UserException.MalformedFile e ) {
+            throw new UserException.MalformedFile(
```

Comment:
A little weird, but I guess worth doing to increase the specificity of the error message.